ClientManagementService/ClientManagementService.Test/ClientCreator.cs

```diff
@@ -0,0 +1,59 @@
+﻿using ClientManagementService.API.DTO;
+using DbClient = ClientManagementService.Infrastructure.Persistence.Entities.Client;
+using DomainClient = ClientManagementService.Domain.Models.Client;
+
+namespace ClientManagementService.Test
+{
+    public static class ClientCreator
+    {
+        public static DbClient GetDbClient(byte[] encryptedPassword)
+        {
+            return new DbClient();

```

Comment:
why is this empty? should populate it with properties.

Reply:
populated